front/src/modules/ui/theme/constants/modal.ts

```diff
@@ -0,0 +1,13 @@
+export const modal = {
+  size: {
+    sm: 300,
+    md: 400,
+    lg: '53%',
+  },
+  padding: {
```

Comment:
Instead of creating new constant, we should create the spacing with `theme.spacing()` function.
I think we could directly use `theme.spacing` in the `switch...case` in the `Modal` component.

Reply:
done this for padding, also change 10 20 30 to 8 16 24 for sm, md, lg padding respectively